CHANGES.rst

```diff
@@ -16,6 +16,10 @@ associations
 
 - Update diagrams to change sloper to detector1. [#4986]
 
+  barshadow
```

Comment:
Not sure it matters, but you might need to remove the extra whitespaces in front of this section title.